src/sidebar/components/test/annotation-header-test.js

```diff
@@ -93,5 +93,56 @@ describe('sidebar.components.annotation-header', function () {
         assert.deepEqual(ctrl.displayName(), 'Bill Jones');
       });
     });
+
+    describe('#thirdPartyUsernameLink', () => {
+      it('returns the thirdPartyUsernameLink if set', () => {
+        var ann = fixtures.defaultAnnotation();
+        var ctrl = $componentController('annotationHeader', {}, {
+          annotation: ann,
+        });
+        assert.deepEqual(ctrl.thirdPartyUsernameLink, 'http://www.example.org/bill');
+      });
+    });
   });
 });
+
+describe('#thirdPartyUsernameLink not set', function () {
```

Comment:
The first parameter to `describe` should usually be a symbol (class, function, method or property). If you are testing a _situation_ (eg. "when the user is not logged in"), use `context` instead. It behaves exactly the same, but it just makes it clearer at a glance what the block tests.